src/components/SipProvider/index.js

```diff
@@ -228,16 +228,20 @@ export default class SipProvider extends React.Component {
 
       // identify call direction
       if (originator === 'local') {
+        const foundUri = rtcRequest.to.toString();
+        const delimiter = foundUri.indexOf(';') || null;
```

Comment:
`delimiter` -> `indexOfDelimiter`? `delimiter` sounds like it's the char itself.